docker-compose/cassandra-3.11/start_cass_311.sh

```diff
@@ -40,5 +40,4 @@ docker-compose up -d cassandra-3
 
 # Bring up the stargate
 
-docker-compose up -d coordinator restapi
-docker-compose up -d coordinator graphqlapi
+docker-compose up -d coordinator restapi coordinator graphqlapi
```

Comment:
Will this work with coordinator listed twice?

Reply:
😳